feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/staking/delegation/proxy/revoke/ConfirmRemoveStakingProxyViewModel.kt

```diff
@@ -133,7 +133,6 @@ class ConfirmRemoveStakingProxyViewModel(
 
             feeMixin.loadFee(
                 this,
-                chain.id,
```

Comment:
expected chain was used only here. But it's confirmation screen and chain here can not be changed.
It seems it was provided incorrectly

Reply:
It was also initially used on SelectSendViewModel untill we for some reason decided not to use loadFee extensions but regular combine
expected chain was introduced for cross chain transfers when origin chain can be changed (get DOT option on Swaps e.g.) to avoid race conditions related to fee loading & chain change